django/contrib/admin/options.py

```diff
@@ -829,6 +829,15 @@ def get_action(self, action):
             description = capfirst(action.replace('_', ' '))
         return func, action, description
 
+    def get_empty_value_display(self):
```

Comment:
Ok, that looks like a good idea. If it's something more complicated than just a string it can always be converted into a `@property`, right?

Would it be useful to also promote this empty value to the fields in `ModelForm`s? Maybe open an additional ticket for that.